### PR TITLE
[Migrations] SQLite/Loki execution

### PR DESCRIPTION
**Pull request by radex**
- Adds execution of needed migrations for Loki and SQLite
  - helper for finding all migration steps for a given version range
  - loki helper methods
  - SQL generation helpers
  - missing fp helper

### CODE REVIEW

**Comment by radex** [comment]
⚠️ don't merge yet

@kokusGr CR please ;)

**Comment by kokusGr** [comment]
LGTM :)

**Comment by radex** [comment]
Test

**Comment by radex** [comment]
test

File: .eslintrc.yml [hunk 1]
```diff
@@ -91,6 +91,7 @@ rules:
     - always
     - excludeArrowFunctions: true
       annotateUndefined: always
+  import/prefer-default-export: 0
```
[the remark this answers is not in the document]
Comment (radex) [reply .eslintrc.yml]: we have the same rule in Purple. This comes up in files like `helpers.js`: https://github.com/Nozbe/WatermelonDB/pull/68/files#diff-57fc1b2445cf88a00fee28b6cd0aedc8R15 where right now I have just one helper, but in the future there might be more, so I don't want to force default export here

**Comment by radex** [comment]
@mobily 🏓

**Pull request merged**